atm.js

```diff
@@ -1,21 +1,96 @@
-var checkingBalance = document.querySelector("div#checking_balance");
-checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
+// #ATM application
+//
+// # Due Friday July 10th
+//
+// We 're going to build an application to keep track of the checking and savings balances. You can play with an example of what a working version will do in the `atm-obfuscate` folder.
+//
+// ## USE jQUERY
+//
+// ###User Stories
+// - As a user, I want to deposit money into one of the bank accounts
+// - As a user, I want to withdraw money from one of the bank accounts
+//   - Make sure the balance in an account can't go negative. If a user tries to withdraw more money than exists in the account, ignore the transaction.
+// - As a user, I want overdraft protection
+//   - What happens when the user wants to withdraw more money from the checking account than is in the account?
+//   - If a withdrawal can be covered by the balances in both accounts, take the balance of the account withdrawn from down to $0 and take the rest of the withdrawal from the other account.
+//   - If the withdrawal amount is more than the combined account balance, ignore it.
+// - As a user, I want the color of my back account to reflect it's balance (there's a CSS class called .zero already written for this!)
+// - Are there ways to refactor your code to make it DRYer or more Object-Oriented?
+//
+// ###Tips
+// * Tackle making your accounts work individually first
+// * Then tackle them working together with overdraft protection
+// * Only start working with the DOM after you have the logic down
+//
+// ###Resources
+// * [What is window.onload for?](https://developer.mozilla.org/en-US/docs/Web/API/GlobalEventHandlers.onload)
+// * [Adding Event Listeners so things happen when you click](https://developer.mozilla.org/en-US/docs/Web/API/EventTarget.addEventListener)
+// * [Adding or removing a CSS class using JS](https://developer.mozilla.org/en-US/docs/Web/API/Element.classList)
+// * [Changing HTML with JavaScript](https://developer.mozilla.org/en-US/docs/Web/API/Element.innerHTML)
 
-var checkingDeposit = function(amount){
-  checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
+var balance = 0;
+var balanceSaving = 0;
 
```

Comment:
im not sure about these variables. Just in that its not modular. Basically you are setting these values at the onset. Imagine once we know about data persistence, and the balance doesn't always start at 0. I think it would be better to get these values from the DOM instead of hardcoding them.